src/views/v2/rsvp-kitchen-sink/success.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Block: RSVP default
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink/default.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 11f9ac7509e98cdef46f9e3a9baaf76e</sub>